src/region/RegionAnnotations.tsx

```diff
@@ -97,13 +98,15 @@ export default class RegionAnnotations extends React.PureComponent<Props, State>
         return (
             <>
                 {/* Layer 1: Saved annotations */}
-                <RegionList
-                    activeId={isCreating ? null : activeAnnotationId}
-                    annotations={annotations}
-                    className="ba-RegionAnnotations-list"
-                    onSelect={this.handleAnnotationActive}
-                    scale={scale}
-                />
+                {isVisible && (
```

Comment:
Should this be handled at a higher level, like in BaseAnnotator or DocumentAnnotator, since it's a global setting? Otherwise we'll have to replicate this logic for all future annotation types.